reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -802,7 +848,11 @@ public Emission getReason() {
 		 * @deprecated to be removed shortly after 3.4.0-RC1. Use {@link #tryEmitValue(Object)} and handle the result.
 		 */
 		@Deprecated
-		void emitValue(@Nullable T value);
+		default void emitValue(@Nullable T value) {
+			emitValue(value, EmitFailureHandler.FAIL_FAST);
+		}
+
+		void emitValue(@Nullable T value, EmitFailureHandler failureHandler);
```

Comment:
missing javadoc. insist on the fact that this should rely on `tryEmitValue` (see https://github.com/reactor/reactor-core/pull/2405#discussion_r496589276)

also, big red warning that the method can throw an `EmissionException` (see https://github.com/reactor/reactor-core/pull/2405#discussion_r496559468)